### PR TITLE
[Concept Entry] Go Math Functions.

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

In the interest of filling out our Go entries, I created a Math Functions concept so we can start producing issues for Go math functions. I included one term so that the list wouldn't be empty when this is merged.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/abs/abs.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Where `result` is the absolute value of `number`.
+
+- The result of `Abs(-Inf)` is `+Inf`
+- The result of `Abs(+Inf)` is `+Inf`
+- The result of `Abs(NaN)` is `NaN`
+
+## Example
+
+The following caluates the absolute value of `-255` and prints out the result:
```
Comment (KTom101) [inline content/go/concepts/math-functions/terms/abs/abs.md:34]:
```suggestion
The following calculates the absolute value of `-255` and prints out the result:
```

File: content/go/concepts/math-functions/math-functions.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+Go supports many different math functions through the `math` package. You must import the `math` package before using these functions, as in the example below:
```
Comment (KTom101) [inline content/go/concepts/math-functions/math-functions.md:15]:
```suggestion
Go supports many different **math functions** through the `math` package. The `math` package must be imported before using these functions, as in the example below:
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski Only a couple of changes from me.

**Comment by SSwiniarski** [comment]
@KTom101, I applied your changes.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski I'll pass this on for a second review

**Review by caupolicandiaz — APPROVED** [review]
@SSwiniarski LGTM. My only comment is that perhaps we should use a different example on the landing page so that we don't have the exact copy in the entry for ABS

**Comment by yangc95** [comment]
@SSwiniarski live links: Go [Math Functions](https://www.codecademy.com/resources/docs/go/math-functions): [abs()](https://www.codecademy.com/resources/docs/go/math-functions/abs)

TY @caupolicandiaz @KTom101

**Pull request merged**